app/src/main/java/co/develhope/meteoapp/network/weather/WeatherService.kt

```diff
@@ -0,0 +1,14 @@
+package co.develhope.meteoapp.network.weather
+
+import co.develhope.meteoapp.network.weather.dto.DaySummary
+import co.develhope.meteoapp.network.weather.dto.WeeklySummary
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface WeatherService {
+    @GET("forecast?hourly=temperature_2m,rain,showers,snowfall,weathercode,windspeed_10m&current_weather=true&timezone=Europe%2FBerlin&start_date=2022-10-03&end_date=2022-10-03")
```

Comment:
Le date dovreste passarle nella funzione in modo da essere dinamiche in base a quelle che servono nella home